frontend/src/Components/MilestonesBarChart.tsx

```diff
@@ -75,7 +75,8 @@ const MilestonesBarChart = ({ data }: { data: ProgramMilestones[] }) => {
                     dataKey={'name'}
                     type="category"
                     width={70}
-                    tick={<YAxisTick />}
+                    tick={YAxisTick}
+                    // tick={<YAxisTick />} //TODO: why is this passed as an element and not a ref
```

Comment:
I had the same thought, but it does indeed say to pass the element in the [docs](https://recharts.org/en-US/api/YAxis#tick)